app/src/main/java/com/github/se/gomeet/ui/mainscreens/events/Events.kt

```diff
@@ -476,18 +476,25 @@ fun EventWidget(
                     Row(
                         verticalAlignment = Alignment.CenterVertically,
                         horizontalArrangement = Arrangement.Center) {
-                          Text(
-                              userName,
-                              style =
-                                  TextStyle(
-                                      fontSize = smallTextSize.sp,
-                                      lineHeight = 24.sp,
-                                      fontFamily = FontFamily(Font(R.font.roboto)),
-                                      fontWeight = FontWeight(700),
-                                      color = MaterialTheme.colorScheme.onBackground,
-                                      letterSpacing = 0.15.sp,
-                                  ),
-                              modifier = Modifier.padding(top = 5.dp).testTag("UserName"))
+                          var username by remember { mutableStateOf<String?>("Loading...") }
```

Comment:
Even with loading screens, I think it's a good idea to have this here just in case the screen decides to finish loading before the username is acquired